lib/cli/index.js

```diff
@@ -1,8 +1,16 @@
 'use strict';
 
+var roadrunner = require('roadrunner');
+roadrunner.load();
+
 // work around misbehaving libraries, so we can correctly cleanup before
 // actually exiting.
-require('capture-exit').captureExit();
+var captureExit = require('capture-exit');
+captureExit.captureExit();
+
+captureExit.onExit(function() {
+  roadrunner.save();
```

Comment:
we may need to specify a name here: https://github.com/kittens/roadrunner/blob/master/index.js#L21